src/patternfly/components/ProgressStepper/examples/ProgressStepper.md

```diff
@@ -530,7 +530,7 @@ cssPrefix: pf-v6-c-progress-stepper
       {{/progress-stepper-step-title}}
     {{/progress-stepper-step-main}}
   {{/progress-stepper-step}}
-  {{#> progress-stepper-step progress-stepper-step--IsInProgressAlt="true" progress-stepper-step--IsCurrent="true"}}
+  {{#> progress-stepper-step progress-stepper-step--IsInProgressAnimate="true" progress-stepper-step--IsCurrent="true"}}
```

Comment:
LGTM though the title for this example is "Basic with Patternfly icons". Should that say something like "Basic in progress" instead?